autofs-5.1.2 - Drop redundant \n in logerr()

From: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>

Line break isn't necessary from the way logerr (and logmsg) is implemented,
and these two probably aren't intentional.

Signed-off-by: Tomohiro Kusumi <kusumi.tomohiro@gmail.com>
Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG       |    1 +
 daemon/module.c |    2 +-
 lib/nss_parse.y |    2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 288d175..99dc245 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -3,6 +3,7 @@ xx/xx/2016 autofs-5.1.3
 - fix release date in CHANGELOG.
 - build: check for clock_gettime in librt.
 - fix compiler warning in try_remount().
+- drop redundant \n in logerr().
 
 15/06/2016 autofs-5.1.2
 =======================
diff --git a/daemon/module.c b/daemon/module.c
index bed8f7a..db01db8 100644
--- a/daemon/module.c
+++ b/daemon/module.c
@@ -34,7 +34,7 @@ int load_autofs4_module(void)
 	 */
 	fp = open_fopen_r("/proc/filesystems");
 	if (!fp) {
-		logerr("cannot open /proc/filesystems\n");
+		logerr("cannot open /proc/filesystems");
 		return 0;
 	}
 
diff --git a/lib/nss_parse.y b/lib/nss_parse.y
index 055e9d7..0721ba1 100644
--- a/lib/nss_parse.y
+++ b/lib/nss_parse.y
@@ -169,7 +169,7 @@ int nsswitch_parse(struct list_head *list)
 
 	nsswitch = open_fopen_r(NSSWITCH_FILE);
 	if (!nsswitch) {
-		logerr("couldn't open %s\n", NSSWITCH_FILE);
+		logerr("couldn't open %s", NSSWITCH_FILE);
 		return 1;
 	}
 
